docs/gadget-devel/metrics.md

```diff
@@ -145,8 +154,7 @@ data source later on) for the map `metrics` defined above. That map is of type `
 `struct metrics_key` as its key and `struct metrics_value` as its value types respectively.
 
 If you look at the `struct metrics_value` definition, you see one field of type `gadget_counter__u32`. This will tell
-IG that this value is meant to be registered as a `counter` of type `__u32`. All members of `struct metrics_value` will
-be registered as `keys` (labels).
+IG that this value is meant to be registered as a `counter` of type `__u32`.
 
 You can fill the map like you usually would (e.g. using `bpf_map_update_elem`, `bpf_map_lookup_elem` and so on), but
 you still need to annotate the data source with key `metrics.collect` and value `true` like so:
```

Comment:
Please mention that dev also still needs to annotate one of the fields as key. It can be either from the fields emitted by the gadget or from the ones enriched by IG, like from the [container-enrichment](https://inspektor-gadget.io/docs/latest/gadget-devel/gadget-ebpf-api/#container-enrichment).

Reply:
Oh right, the metrics.collect is a different one.